build-tools/packages/build-cli/src/test/init.ts

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import path from "node:path";
+
+import { _dirname } from "./dirname.cjs";
+
+export const packageRootPath = path.resolve(_dirname, "../..");
+
+/**
+ * Absolute path to the test data. It's rooted two directories up because the tests get executed from lib/.
+ */
+export const testDataPath = path.resolve(_dirname, packageRootPath, "src/test/data");
+
+/**
+ * Absolute path to the test repo.
+ */
+export const testRepoRoot = path.join(testDataPath, "testRepo");
```

Comment:
```suggestion
/**
 * Absolute path to the test data.
 */
export const testDataPath = path.resolve(
	_dirname,
	"../../../build-infrastructure/src/test/data",
);

/**
 * Absolute path to the test repo.
 */
export const testRepoRoot = path.join(testDataPath, "testRepo");

```

This along with the dirname change should fix the test failures.